kobo/apps/openrosa/apps/logger/models/attachment.py

```diff
@@ -43,6 +45,9 @@ def get_queryset(self):
 
 
 class Attachment(AbstractTimeStampedModel, AudioTranscodingMixin):
+    # Mimic KpiUidField behaviour with _null=True until TASK-1534 is completed
+    # TODO: update uid to generate_unique_id('att') for new attachments
```

Comment:
This is  (already) done by KpiUidField.
TODO should be, remove `_null=True` when TASK-1534 is complete